<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_picture">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>611</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>705</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_loadFromFile">
      <property name="text">
       <string>loadFromFile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_loadFromWeb">
      <property name="text">
       <string>loadFromWeb</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
